src/ontology/mondo-ingest.Makefile

```diff
@@ -553,12 +553,11 @@ $(REPORTDIR)/%.subclass.confirmed.robot.tsv $(REPORTDIR)/%.subclass.added.robot.
 ##################################
 ## Externally managed content ####
 ##################################
+# General
+update-externally-managed-content: external-content-nord external-content-nando external-content-ordo external-content-omim
```

Comment:
## General changes I slipped in
Sorry about this. I have a number of small PRs I want to do, little time, so I slipped some things in here.

0. Didn't do: I'm considering in another PR to move NanDO next to Nord. I want the "wholly externally managed" stuff grouped next to each other.
1. Added comments for each subsection of 'externally managed content': general, nord, nando, ordo, omim
2. Moved some things that were out of position. Moved the 'general' stuff together, and to the top. Moved goals that were for specific sources together.
3. Delete: There were 2 goals for `external-content-nord`. Now there is just one.
4. Added a missing `mkdir -p $(EXTERNAL_CONTENT_DIR)` from one of the goals for a given source. Now each of the 4 sources in "external" have this line. In a separate PR, I want to refactor this to an 'order-only prerequisite' `|  $(EXTERNAL_CONTENT_DIR)` on each of the goals, as that is cleaner and a more standard pattern.

Reply:
I've only created the ones for `ordo` (in one of the ORDO subsets PRs), and `omim` (this PR). Nico created the other ones in individual PRs.